server/index.js

```diff
@@ -20,26 +20,42 @@ app.get('/reviews', (req, res) => {
       throw new Error(err);
     } else {
       // eslint-disable-next-line no-console
-      console.log('result! ', result);
       res.sendStatus(200);
       return result;
     }
   });
 });
 
-app.get('/review', (req, res) => {
-  const productId = req.body.product_id;
-  Reviews.findOne({ product_id: productId }, (err, result) => {
+app.get('/review/:productId', (req, res) => {
+  const starCount = {};
+  const reviewData = {};
+  const { productId } = req.params;
+  Reviews.findOne({ product_id: productId.toUpperCase() }, (err, result) => {
     if (err) {
       // eslint-disable-next-line no-console
```

Comment:
for this i believe You can edit eslint settings to ignore no-console error. this way You won't need to put comments everywhere you have console.log

Reply:
I agree, although I feel I may miss them then. I'm actually curious if there is a way in webpack when compiling your bundle.js for production to remove all mentions of console logs. Going to look into that. Thanks for the feedback.